thirdparty/mupdf/mupdf_cbz_chapter_support.patch

```diff
@@ -0,0 +1,120 @@
+commit bfae5958ba7ebeb7bb58c0a0e56bb6e521ef6d9f
+Author: Marcel Röthke <marcel@roethke.info>
+Date:   Sun Nov 13 22:10:06 2022 +0100
+
+    add support for chapters in cbz
```

Comment:
An actual description of how this actually works and how it expects CBZ to be organized would be helpful.